src/lib/y2users/user.rb

```diff
@@ -86,6 +86,13 @@ class User < ConfigElement
     # @return [Password]
     attr_accessor :password
 
+    # Authorized keys
+    #
+    # TODO: use an specialized SSH public keys collection
```

Comment:
Why is a collection needed?

Reply:
As discussed _offline_, because in the feature we might end up needing a collection to manage them when working in an installed system. But, as agreed, let's drop the TODO by now since for the current code base an Array is more than enough.

Thanks!